gateway/README.md

```diff
@@ -31,31 +31,9 @@ The first step will be to determine which gateway modes and functions you wish t
 
 ### Create secrets
 
-There are a number of ways that Kubernetes secrets can be managed. If you do not have an existing external secret manager for your Kubernetes clusters, we recommend creating a secret manually outside of the context of this chart, which will then be referenced in your `values.yaml` file.
+There are a number of ways that Kubernetes secrets can be managed. If you do not have an existing external secret manager for your Kubernetes clusters, you can create secrets by using the `appSecrets` section of the `values.yaml` file.
```

Comment:
might be a good idea to reiterate point about this not being best practice here too